src/styles/global.scss

```diff
@@ -14,6 +14,86 @@
   }
 }
 
+html.is-extension {
+  width: $extension-width;
+  height: $extension-height;
+
+  // fixes issue with extension not rendering properly
+  ion-tabs,
+  .tabs-inner,
+  ion-router-outlet,
+  .ion-page {
```

Comment:
There is no such class in whole app.

Reply:
It's an ionic class, added to ionic pages by ionic